src/playlist.js

```diff
@@ -305,6 +305,39 @@ const calculateExpiredTime = function(playlist, expiredSync, segmentSync) {
 
     return segmentSync.time - sumDurations(playlist, syncIndex, 0);
   }
+  return null;
+};
+
+/**
+ * Calculates the playlist end time or seekable end time depends on using
+ * safe live end or not
+ *
+ * @param {Object} playlist a media playlist object
+ * @param {Object|null} useSafeLiveEnd a boolean value if true then live playlists
```

Comment:
If it is a boolean maybe change the param from `Object|null` to `Boolean|false`

And the description could probably be something like `Whether or not the playlist end calculation should consider the safe live end (truncate the playlist end by three segments). This is normally used for calculating the end of the playlist's seekable range.`